generic-collection.js

```diff
@@ -5,6 +5,8 @@ function GenericCollection() {
     throw new Error("Can't construct. GenericCollection is a mixin.");
 }
 
+GenericCollection.EmptyArray = Object.freeze([]);
```

Comment:
I agree, but if you’ve got FRB, you’ve got shims. I’m trying to get rid of shims in the v2 branch, though. If you go that way, isn’t too wasteful to just have a frozen array per module or to have a module with the frozen array.